moto/sesv2/urls.py

```diff
@@ -0,0 +1,14 @@
+"""sesv2 base URL and path."""
+from .responses import SESV2Response
+
+url_bases = [
+    r"https?://email\.(.+)\.amazonaws\.com",
+]
+
+
+response = SESV2Response()
+
+
+url_paths = {
+    "{0}/v2/.*$": response.dispatch,
```

Comment:
```suggestion
    "{0}/v2/email/outbound-emails$": response.dispatch,
    "{0}/v2/email/contact-lists/(?P<name>[^/]+)/contacts$": response.dispatch,
```
The URL's need to be specified explicitly, then it will work